src/components/loader-bar/loader-bar.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { ComponentStory } from "@storybook/react";
+import LoaderBar from ".";
+
+export const DefaultStory: ComponentStory<typeof LoaderBar> = (args) => (
+  <LoaderBar mt={2} {...args} />
+);
+DefaultStory.storyName = "default";
```

Comment:
I assume this file was created to refactor the `stories.mdx` stories to typescript? If so we will need to move all the stories from the `stories.mdx` file to this new file, then import the stories back into `stories.mdx`.